### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by jk257** [Jyers/Marlin#157] [Q&A] [answered]
I feel like an incredible idiot asking this but what are the two center status icons for that read in %?

Of the 6 I get Nozzle Temp, Bed Temp, Part Cooling Fan Speed and Z-Offset but unfamiliar with the other two. Something to do with Extruder and Board Fan?? I tried finding documentation here and on marlin website and googled for V2 icons but no dice.

**Comment by DragRedSim** [comment, marked as answer]
Pretty sure they're Print Speed and Flow Rate, respectively.

**Reply by Schnabulation** [reply]
^ correct

**Comment by jk257** [comment]
Thank you very much! Now I can do my face plam 🤦🏻‍♂️ and sit in the corner. 🤣

**Reply by Schnabulation** [reply]
No worries - everybody starts somewhere ;-)
